airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -65,6 +65,7 @@
   "connection.dbtCloudJobs.genericError": "There was an error communicating with dbt Cloud.",
   "connection.dbtCloudJobs.explanation": "After an Airbyte sync job has completed, the following jobs will run",
   "connection.dbtCloudJobs.noJobs": "No transformations",
+  "connection.dbtCloudJobs.noJobsFoundForAccount": "No transformations found for this dbt Cloud account",
```

Comment:
Maybe "no jobs found for this dbt Cloud account"? Seems like we should use dbt's terminology here, because that's what users will likely need to cross-reference to fix issues.